internal/binutils/binutils_test.go

```diff
@@ -218,6 +218,9 @@ func findSymbol(syms []*plugin.Sym, name string) *plugin.Sym {
 }
 
 func TestObjFile(t *testing.T) {
+	// If this test fails, check the address for main function in testdata/exe_linux_64
+	// using the command 'nm -n '. Update the hard coded addresses below to match
```

Comment:
"hard coded" -> "hardcoded"

Reply:
Not done?

Reply:
Sorry, i am not very familiar with the suggested changes workflow in Github. 
I misinterpreted the UI and did the wrong thing.